scripts/bandwidth_throttling.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -o pipefail
+set -e
+
+######################################
+# Simulating Poor Network Connectivity
+######################################
+enable_bandwidth_throttling() {
+    # run inside sudo
+    sudo sh <<SCRIPT
```

Comment:
How will this work on the default macOS configuration that requires a password be entered to run anything via `sudo`? What will happen when I run the object server tests on my local machine?